openmetadata-ui/src/main/resources/ui/playwright/e2e/VersionPages/EntityVersionPages.spec.ts

```diff
@@ -205,9 +210,8 @@ test.describe('Entity Version pages', () => {
           type: 'Users',
         });
 
-        const versionDetailResponse = page.waitForResponse(`**/versions/0.3`);
```

Comment:
Why are we removing the wait on specific API calls? They were added there for a reason to verify the version change as well.